### ISSUE TITLE
Numbers are displayed strangely

### ISSUE THREAD

**Issue by ms0ur** [Team-Inceptus/Novaconomy#71]
### Bug Type

Other Unexpected Behavior

### Minecraft Version

1.19.4

### Description

Numbers greater than 999, for example 10,000, are displayed very strangely. 
![image](https://user-images.githubusercontent.com/73832732/229373551-0a6480fa-3eda-41c9-aa5c-7691ce2b0da3.png)
![image](https://user-images.githubusercontent.com/73832732/229373560-cbb30b84-536c-4a13-8b90-b17b44115391.png)

### Steps to Reproduce

1. Change lang to RU in config.(in config: `Language: ru`)
1): 1.Run `/bal` . 2.Hover on your balance. 3. ![image](https://user-images.githubusercontent.com/73832732/229373740-277041fe-06fd-4c8f-bc48-c4a601f5cd4f.png)
2): 1. Run `/econ addbal <currency> <player> 10000`. 2. ![image](https://user-images.githubusercontent.com/73832732/229373752-8c65e031-e9f9-4a3c-ac57-a6425c709d7e.png)

### Expected Behavior

I think it should be something like `10,000` or `10000` instead of `10<something strange>000`

### Actual Behavior

Numbers greater than 999, for example 10,000, are displayed very strangely. 
![image](https://user-images.githubusercontent.com/73832732/229373551-0a6480fa-3eda-41c9-aa5c-7691ce2b0da3.png)
![image](https://user-images.githubusercontent.com/73832732/229373560-cbb30b84-536c-4a13-8b90-b17b44115391.png)

### Additional Info

Server version: git-Purpur-1952 (MC: 1.19.4)
Plugin version: latest from spigotmc

**Comment by GamerCoder215** [comment]
It seems some of the formatting issues for numbers don't respect the full characters of the Russian language. I'll work on a fix for v1.7.1.

**Comment by GamerCoder215** [comment]
I did some more research and here is my conclusion.

Cyrillic (or Russian, not sure if there's a difference) numeral formatting uses spaces between the numbers. It uses commas to denote a decimal instead of `.` when using numbers, unlike most Latin-based languages like English.

For example:
```java
System.out.printf(new Locale("ru"), "%,.5f", 123456789.123456789);

/* Output:
123 456 789,12346
*/
```
The Java `format` function parses these spaces as non-breaking spaces (hence the name `nbsp`) instead of regular spaces.

The issue will be marked as resolved when v1.7.1 releases.
